eoscostarica/eos-rate
# feat(demux): add bpjson crawling and storing support for all bps

### GH Issue

#149 
#111 
#142 

### Steps to test

1. Verify a non existent URL
1. Use fab icon on bottom right corner to toggle the compare tool.
1. Data from graphql should load just like it used to.

### Checklist
